docs/table_design/Data_distribution.md

```diff
@@ -151,6 +159,44 @@ AGGREGATE KEY(site_id, city_code, user_name)
 DISTRIBUTED BY HASH(site_id,city_code);
 ```
 
+#### Random bucketing (since v3.1)
+
+For data in a partition, StarRocks distributes the data randomly across all buckets, which is not based on specific column values. Additionally, there is no need to set bucketing columns during table creation, simplifying the CREATE TABLE statement. It is suitable for scenarios with relatively small data sizes and requirement for high query performance.
+
+However, note that random bucketing may not be suitable for scenarios that involve querying and aggregating based on specific columns. In such cases, hash bucketing may be more appropriate as it can store similar data in the same bucket, facilitating data access and processing.
+
+**Precautions**
+- You can not use random bucketing to create a Primary Key table, a Unique Key table, or an aggregated table.
+- You can not specify a table bucketed randomly to belong to a Colocation Group.
+- Spark Load can not be used to load data into tables bucketed randomly.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/f529851a-d9cb-4022-86cb-5f5107cd88c7)
see previous comments:
1. “can not" -> "cannot"
2. add doc links